universal-application-tool-0.0.1/app/export/CsvExporter.java

```diff
@@ -0,0 +1,9 @@
+package export;
+
+import java.io.Writer;
+import models.Applicant;
+
+public class CsvExporter implements Exporter {
+  @Override
+  public void export(Applicant a, Writer w) {}
```

Comment:
nit: please avoid single-letter var names except for extremely common idioms like `i` for an index or `e` for an exception

Reply:
Sure, that's the go style creeping back in.

Reply:
Ah Rob Pike, hater of syntax highlighting and autocomplete